packer/scripts/create-kube-certs.sh

```diff
@@ -2,6 +2,8 @@
 
 set -o errexit -o xtrace
 
+export PATH="${PATH}:/usr/local/bin/"
```

Comment:
Wrong packer; the Jenkins master doesn't get that, so `/usr/local/bin/` isn't in `/etc/login.defs`.